core/src/main/resources/i18n/displayStrings_tr.properties

```diff
@@ -0,0 +1,3345 @@
+shared.makerTxFee=Yapıcı: {0}
+shared.takerTxFee=Alıcı: {0}
+shared.iConfirm=Onaylıyorum
+shared.openURL={0}'i aç
+shared.fiat=Fiat
+shared.crypto=Kripto
+shared.traditional=Nakit
+shared.otherAssets=diğer varlıklar
+shared.other=Diğer
+shared.all=Hepsi
+shared.edit=Düzenle
+shared.advancedOptions=Gelişmiş seçenekler
+shared.interval=Aralık
+shared.actions=Eylemler
+shared.buyerUpperCase=Alıcı
+shared.sellerUpperCase=Satıcı
+shared.new=YENİ
+shared.learnMore=Daha fazla bilgi edin
+shared.dismiss=Kapat
+shared.selectedArbitrator=Seçilen hakem
+shared.selectedMediator=Seçilen arabulucu
+shared.selectedRefundAgent=Seçilen hakem
+shared.mediator=Arabulucu
+shared.arbitrator=Hakem
+shared.refundAgent=Hakem
+shared.refundAgentForSupportStaff=İade temsilcisi
+shared.delayedPayoutTxId=Gecikmiş ödeme işlem kimliği
+shared.delayedPayoutTxReceiverAddress=Gecikmiş ödeme işlemi gönderildi
+shared.unconfirmedTransactionsLimitReached=Şu anda çok fazla onaylanmamış işleminiz var. Lütfen daha sonra tekrar deneyin.
+shared.numItemsLabel=Girdi sayısı: {0}
+shared.filter=Filtrele
+shared.enabled=Etkin
+shared.pending=Beklemede
+shared.me=Ben
+shared.maker=Yapıcı
+shared.taker=Alıcı
+
+
+####################################################################
+# UI views
+####################################################################
+
+####################################################################
+# MainView
+####################################################################
+
+mainView.menu.market=Piyasa
+mainView.menu.buy=Satın Al
+mainView.menu.sell=Sat
+mainView.menu.portfolio=Portföy
+mainView.menu.funds=Fonlar
+mainView.menu.support=Destek
+mainView.menu.settings=Ayarlar
+mainView.menu.account=Hesap
+
+mainView.marketPriceWithProvider.label=Fiyat {0} tarafından
+mainView.marketPrice.havenoInternalPrice=Son Haveno işlem fiyatı
+mainView.marketPrice.tooltip.havenoInternalPrice=Harici fiyat besleme sağlayıcılarından piyasa fiyatı yok.\n\
+  Görüntülenen fiyat, o para birimi için en son Haveno işlem fiyatıdır.
+mainView.marketPrice.tooltip={0}{1} Piyasası\nSon güncelleme: {2}\nSağlayıcı düğüm URL'si: {3}
+mainView.balance.available=Bakiye
+mainView.balance.reserved=Tekliflerde ayrılmış
+mainView.balance.pending=Beklemedeki bakiye
+mainView.balance.reserved.short=Ayrılmış
+mainView.balance.pending.short=Beklemede
+
+mainView.footer.usingTor=(Tor üzerinden)
+mainView.footer.localhostMoneroNode=(localhost)
+mainView.footer.clearnet=(clearnet üzerinden)
+mainView.footer.xmrInfo={0} {1}
+mainView.footer.xmrFeeRate=/ Ücret oranı: {0} sat/vB
+mainView.footer.xmrInfo.initializing=Haveno ağına bağlanıyor
+mainView.footer.xmrInfo.synchronizingWith={0} ile senkronize ediliyor blok: {1} / {2}
+mainView.footer.xmrInfo.connectedTo={0}'a bağlandı blok {1}
+mainView.footer.xmrInfo.synchronizingWalletWith={0} ile cüzdan senkronize ediliyor blok: {1} / {2}
+mainView.footer.xmrInfo.syncedWith={0} ile senkronize edildi blok {1}

```

Comment:
In your screenshot it says: ``Monero Stagenet (Tor üzerinden) ile senkonize edildi`` but instead it should be ``Monero Stagenet ile (Tor üzerinden) senkonize edildi``